conf/docker-aio/c7.dockerfile

```diff
@@ -1,17 +1,21 @@
 FROM centos:7
 # OS dependencies
-RUN yum install -y java-1.8.0-openjdk-headless postgresql-server sudo epel-release unzip perl curl
+RUN yum install -y java-1.8.0-openjdk-headless postgresql-server sudo epel-release unzip perl curl httpd
 RUN yum install -y jq
 
 # copy and unpack dependencies (solr, glassfish)
 COPY dv /tmp/dv
 COPY testdata/schema.xml /tmp/dv
 COPY testdata/solrconfig.xml /tmp/dv
+# IPv6 and localhost appears to be related to some of the intermittant connection issues
+COPY disableipv6.conf /etc/sysctl.d/
+RUN rm /etc/httpd/conf/*
+COPY testdata/httpd.conf /etc/httpd/conf 
 RUN cd /opt ; tar zxf /tmp/dv/deps/solr-7.2.1dv.tgz 
 RUN cd /opt ; tar zxf /tmp/dv/deps/glassfish4dv.tgz
+COPY domain-restmonitor.xml /opt/glassfish4/glassfish/domains/domain1/config/domain.xml
```

Comment:
If we are overwriting `domain.xml` doesn't that defeat the purpose of running all the `asadmin` commands in the installer? In the bad old days of DVN 3 we would overwrite `domain.xml` like this and we have since invested a lot of effort into figuring out all `asadmin` syntax needed to avoid this.